.github/workflows/ci.yml

```diff
@@ -44,6 +44,7 @@ jobs:
         run: |
           pip install -U "pip==20.2.4" wheel setuptools
           pip install ${{ matrix.django-version }}
+          pip install swapper@git+git://github.com/openwisp/django-swappable-models.git@master
```

Comment:
I added this here because openwisp-users is installing swapper v1.2 and when installing openwisp-radius it is not updating the swapper.